controllers/scopeinstance_controller.go

```diff
@@ -59,6 +59,8 @@ const (
 
 	// generateNames are used to track each binding we create for a single scopeTemplate
 	clusterRoleBindingGenerateKey = "operators.coreos.io/generateName"
+
+	scopeInstanceControllerFieldOwner = "scopeinstance-controller"
```

Comment:
```suggestion
	controllerFieldOwner = "scopeinstance-controller"
```
Shorten the variable name and group it with the one above.

Reply:
Should be resolved with commit https://github.com/awgreene/oria-operator/pull/24/commits/adc34c08f252eb9c231f409898d6c84b07678d98